kolyanTask/task.js

```diff
@@ -0,0 +1,62 @@
+const AIRCRAFT_TYPES = {
+    type1: `plane`,
```

Comment:
```suggestion
    plane: 'plane',
```
use single-quotes for constant strings and human readable names for keys